packages/orm/src/EntityManager.ts

```diff
@@ -2171,7 +2171,7 @@ export class EntityManager<C = unknown, Entity extends EntityW = EntityW, TX ext
 
   /** Creates a new `type` and marks it as loaded, i.e. we know its collections are all safe to access in memory. */
   #doCreate<T extends EntityW>(type: EntityConstructor<T>, opts: any, partial: boolean): T {
-    const entity = new (asInternalCstr(type))(this, true);
+    const entity = newEntity(this, type, true);
```

Comment:
There are only two places in `EM` that called `new`, and both are going through `newEntity` now